lib/init.coffee

```diff
@@ -10,10 +10,13 @@ checkedAppend = (parameters, opt, args) ->
     parameters.push opt
     parameters.push(args...)
 
-makeParameters = (globals, ignore, file) ->
+makeParameters = (globals, ignore, std, file) ->
   parameters = ['-', '--no-color', '--codes', '--ranges', "--filename=" + file]
   checkedAppend parameters, '--globals', globals
   checkedAppend parameters, '--ignore', ignore
+  if std.length > 0
+    parameters.push '--std'
+    parameters.push(std.join '+')
```

Comment:
You are right, that sounds better, alternatively I could make the default value of `standard` to be `_G` which is the default value